ogn/parser/parse.py

```diff
@@ -6,16 +6,18 @@
 from ogn.parser.exceptions import AprsParseError, OgnParseError
 
 
-def parse_aprs(message, reference_date=None):
+def parse_aprs(message, reference_date=None, reference_time=None):
     if reference_date is None:
```

Comment:
If `reference_time` is set, but `reference_date == None`, the `reference_time` is ignored.
I would split those checks:
```
if reference_date is None:
    reference_date = datetime.utcnow().date()
if reference_time is None:
    reference_time = datetime.utcnow().time()
```